nle/env/base.py

```diff
@@ -348,7 +352,7 @@ def _in_moveloop(self, observation):
         program_state = observation[self._program_state_index]
         return program_state[3]  # in_moveloop
 
-    def reset(self):
+    def reset(self, *args, **kwargs):
```

Comment:
I would slightly prefer not to `*args, **kwargs` this but to spell out whatever legal argument we expect -- in this case, only `wizkid`?